herbert_core/utilities/strings/stringmatchi_log.m

```diff
@@ -0,0 +1,73 @@
+function status = stringmatchi_log (str, strcell, exact)
+% Find matches of a character vector to abbreviations of strings, ignoring case
+%
+% Returns a logical array of exact matches of a character vector to a cell array
+% of strings, if there are any.
+% If there are no exact matches, then returns a logical array where the 
+% character vector is an abbreviation of the contents of the cell array.
+%
+%   >> status = stringmatchi_log (str, strcell)
+%   >> status = stringmatchi_log (str, strcell, exact)      % option: logical 0 or 1
+%   >> status = stringmatchi_log (str, strcell, 'exact')    % option: character string
```

Comment:
I would pick one or the other.

Reply:
I agree that simplicity is good. However, in the spirit of so many things going to #1485, and this option is being used in a rejigged development to simplify some of our parsing routines, the topic of another couple of tickets to be rationalised, I'd prefer to leave as it is.